cypress/integration/AnnualPlan/ContentBlockAPLink/contentBlock.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="Cypress" />
+
+import { Given, Then } from "cypress-cucumber-preprocessor/steps";
+
+Then('the PDF download link should appear', () => {
+    cy.get(`.ap-file-block`).should('be.visible');
+});
+
+And('a PDF icon should be displayed with a text {string}', (linkName) => {
+    cy.get(`.ap-file-block a[target='_blank']`).should('be.visible');
+    cy.get(`.ap-file-block a`).should('have.text', linkName);
+});
+
+And('clicking on the link will open a new window', () => {
+    cy.get(`.ap-file-block a`).should('have.attr', "target", "_blank");
+
+});
+
+Then('the PDF download link is not displayed', () => {
+    cy.get(`.ap-file-block`).should('not.be.visible');
+});
+
+When('user clicks on top {string} button', (hrefAttr) => {
+    cy.get(`.ap-file-block a`).click({ force: true });
```

Comment:
If you use the trigger() function instead of clikck(), with the added option of not following the redirect, the page will not open a new tab - please replace with
_cy.get("div[class='ap-file-block pdf']").find('a').trigger('click', {followRedirect:false});_

Reply:
fixed it